query/src/sql/opt/cascades/explore_rules.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::sql::opt::RuleSet;
+
+pub fn get_explore_rule_set() -> RuleSet {
+    RuleSet::create_with_ids(vec![]).unwrap()
+}
+
+#[cfg(test)]
```

Comment:
:+1: 

The test style has changed, we place them to query/tests/it/sql

Reply:
I'll migrate them later.